docs/notes/versions.rst

```diff
@@ -49,6 +49,61 @@ The following table shows the minimum version required of the Fast DDS build sys
 
 .. tabs::
 
+    .. group-tab:: 2.14.x
+
+        .. list-table::
+
+            * - **CMake**
+              - 3.20
+
+        .. list-table::
+            :header-rows: 1
+
+            * - OS \ Architecture
+              - amd64
+              - amd32
+              - arm64
+            * - Ubuntu Jammy (22.04)
+              - Tier 1: GCC 11.4 |br|
+                Tier 3: Clang 15
+              - **───**
+              - Tier 1: GCC 11.4 |br|
+                Tier 3: Clang 15
+            * - Ubuntu Focal (20.04)
+              - Tier 3: GCC 9
+              - **───**
+              - Tier 3: GCC 9
+            * - MacOS Mojave (10.14)
+              - Tier 1: Clang 15
+              - **───**
+              - **───**
+            * - Windows 10
+              - Tier 1: MSVC v142 (Visual Studio 2019) |br|
+                Tier 2: MSVC v141 (Visual Studio 2017)
+              - Tier 2: MSVC v142 (Visual Studio 2019), |br|
+                Tier 2: MSVC v141 (Visual Studio 2017)
```

Comment:
It seems like someone left a comma in one of the cases but not in the other.
I would suggest removing the comma in all cases, not just 2.14.x

```suggestion
              - Tier 1: MSVC v142 (Visual Studio 2019) |br|
                Tier 2: MSVC v141 (Visual Studio 2017)
              - Tier 2: MSVC v142 (Visual Studio 2019) |br|
                Tier 2: MSVC v141 (Visual Studio 2017)
```